src/Plutarch/Extra/Value.hs

```diff
@@ -486,6 +488,34 @@ phasOnlyOneTokenOfCurrencySymbol = phoistAcyclic $
     psymbolValueOf # cs # vs #== 1
       #&& (plength #$ pto $ pto $ pto vs) #== 1
 
+{- | Returns 'PTrue' if the argument 'PValue' contains /exactly/
+  one token of the argument 'PAssetClass'.
+
+ @since 3.9.1
+-}
+phasOneTokenOfAssetClass ::
+  forall
+    (tag :: Symbol)
+    (keys :: KeyGuarantees)
+    (amounts :: AmountGuarantees)
+    (s :: S).
+  Term s (PAssetClass tag :--> PValue keys amounts :--> PBool)
+phasOneTokenOfAssetClass = phoistAcyclic $
+  plam $ \cls v ->
+    passetClassValueOf # cls # v #== 1
+
+{- | Returns 'PTrue' if the argument 'PValue' contains /exactly/ one
+ token of the argument 'PCurrencySymbol'.
```

Comment:
got it, will add a note.
An alternative name suggestion would also be welcome

Reply:
Do we have to have `AssetClass` spelled out? I think it can be shortened into `AC`. I think there are several things we can shorten as well--like `CurrencySymbol` will be `CS`, token will be `T`. This way we can have function name more descriptive about the action instead of what it takes. 

With this naming this function will be `pACContainsOneT` or something.

Reply:
Also, I think we can even drop `AssetClass` from the name as it is already described in the type.

Reply:
I think the conventions (in Liqwid) have been `cls` for asset class, `cs` or `symbol` for currency symbol, `tn` or `name` for token name.

I'm fine with whatever, but once we decide we should add these to the style guide for consistency.